package.json

```diff
@@ -141,6 +141,7 @@
     "@types/json-stable-stringify": "^1.0.32",
     "@types/lodash.clonedeep": "^4.5.4",
     "@types/node-forge": "^0.9.0",
+    "@types/query-string": "^6.3.0",
```

Comment:
Was this intended? AFAICT this isn't related to your changes.

Reply:
It's not directly related to my changes but during my merge I noticed my IDE was complaining about missing types for this. 

I'm not sure why eslint/typescript isn't complaining about it but I figured it couldn't hurt to add.

Probably bad form to sneak this into the PR... Do you want me remove it?

Reply:
Ah, ok. I think it would probably be preferable to open a separate PR, but I won't block this one on it.

Reply:
On top of it not being great git etiquette it was also causing merge conflicts so I took that as a sign to not do this in this PR. Doesn't seem to be causing problems anyways.